packages/stark-ui/src/modules/svg-view-box/directives/svg-view-box.directive.spec.ts

```diff
@@ -106,14 +104,15 @@ describe("SvgViewBoxDirective", () => {
 			initializeComponentFixture();
 		});
 
-		it("should keep the viewBox attribute as is", () => {
+		// tslint:disable-next-line:no-identical-functions
```

Comment:
Maybe this is not needed if yoy re-use the `DEFAULT_VIEW_BOX_SIZE` constant as I mentioned in the previous remark ;)

Reply:
It is, because I also changed it in the test above, but I did move it to the top of the file.

Reply:
oops... I missed that, sorry 😉